canonical-web-and-design/multipass.run
# Convert site to Flask

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2145, for https://github.com/canonical-web-and-design/web-squad/issues/2148

QA
--

- `./run clean && ./run`, go to `http://localhost:8026` and see the site, check it works
- `./run build && docker build -t multipass . && docker run -ti -p 8126:80 multipass`, then go to http://localhost:8126, see the site
- `./run test`, check it works
- check the CI tests pass
